components/Icons/SearchMenu.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const Icon = ({ size, fill, ...props }) => (
+  <svg width={size} height={size} viewBox='0 0 26 18' {...props}>
+    <path
+      fillRule='evenodd'
+      clipRule='evenodd'
```

Comment:
clip rule is unnecessary:
```suggestion
```

I generally would also recommend running icons through [SVGOMG](https://jakearchibald.github.io/svgomg/) (would reduce the number of paths elements in this cases).

Reply:
done